app/models/station.rb

```diff
@@ -41,4 +41,42 @@ def self.minimum_dock_count
     minimum(:dock_count)
   end
 
+  def started_rides_count
+    trips_started.count
+  end
+
+  def ended_rides_count
+    trips_ended.count
+  end
+
+  def frequent_destination
+    station_id = trips_started.group("end_station_id").order("count_id DESC").count(:id).first[0]
+    Station.find(station_id)
+  end
+
+  def frequent_destination_name
+    frequent_destination.name
+  end
```

Comment:
Like that you're separating pulling out the station from pulling out the name of that station. 👍 

I could potentially reuse `frequent_destination` elsewhere.